src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -81,7 +81,11 @@
       <f:textbox/>
     </f:entry>
 
-    <f:descriptorList title="${%Node Properties}" descriptors="${h.getNodePropertyDescriptors(descriptor.clazz)}" field="nodeProperties" />
+    <!-- descriptorList is not working somehow, repeatableProperty has same functionality, just ui changes a little -->
+    <f:entry title="${%Tool Locations}">
+        <f:repeatableProperty field="nodeProperties" noAddButton="false" />
```

Comment:
this gives a weird UI with 2 clicks to get to the data

![screenshot 2018-05-22 15 14 02](https://user-images.githubusercontent.com/23651/40364549-ecc76bde-5dd2-11e8-87b2-cb0dd521f923.png)

Reply:
I'd better stick to the existing one.